ts/components/DebugPrettyPrint.tsx

```diff
@@ -1,71 +0,0 @@
-import {
```

Comment:
This is an useful component for debugging: it prints a JSON content. It's not used in the app directly, but could be used during the development.